webapp/client/src/components/FormFieldDropDown.js

```diff
@@ -52,21 +43,21 @@ function FormFieldDropDown({
       hideLabel
       render={() => (
         <DropDown>
-          <FieldLabelForSeparatedFields {...{ label, fieldId, details }} />
+          <FieldLabel {...{ label, fieldId, details }} />
           <select
             id={fieldId}
             className="custom-select steuerlotse-select"
             input_req_err_msg="Bundesland auswählen"
             name={fieldId}
-            value={preselectedValue}
+            defaultValue={preselectedValue}
             required={required}
             autoFocus={autofocus || Boolean(errors.length)}
             onBlur={onChangeHandler}
             onChange={onChangeHandler}
           >
-            {defaultValue && <option value="">{defaultValue}</option>}
-            {values.map((value) => (
-              <option value={value[0]}>{value[1]}</option>
+            {defaultOption && <option value="">{defaultOption}</option>}
+            {options.map((option) => (
+              <option value={option[0]}>{option[1]}</option>
```

Comment:
Hm, I think this would be more readable if we made the `option`s into objects instead of an arrays? Something like:
```js
{
  value: "by",
  displayName: "Bayern"
}
```

Reply:
Great idea 👍